example/example.ino

```diff
@@ -6,15 +6,14 @@ void setup() {
 
 int c = 0;
 void loop() {
-    if (c == 0) {
+    if (Serial.read() == 'a') {
         int a[12][2];
         for (int i = 0; i < 12; i++) {
             a[i][0] = random(100);
             a[i][1] = i;
         }
-  
-        //  sorter::sort_max(a);  // Set two first indexes in array to max values
-        //  sorter::sort_min(a);  // Set two first indexes in array to min values
+        // sorter::sort_max(a);  // Set two first indexes in array to max values
+          sorter::sort_min(a);  // Set two first indexes in array to min values
```

Comment:
Please remove 2 spaces before `sorter::sort_min(a);` so that it's on the same indentation level as the rest of the code.